src/Sulu/Bundle/CustomUrlBundle/EventListener/CustomUrlSerializeEventSubscriber.php

```diff
@@ -61,23 +63,42 @@ public static function getSubscribedEvents()
     public function onPostSerialize(ObjectEvent $event)
     {
         $customUrl = $event->getObject();
+
+        /** @var SerializationVisitorInterface $visitor */
         $visitor = $event->getVisitor();
 
         if (!$customUrl instanceof CustomUrlDocument) {
             return;
         }
 
         if (null !== $customUrl->getTargetDocument()) {
-            $visitor->addData('targetTitle', $customUrl->getTargetDocument()->getTitle());
```

Comment:
Does `addData` not exist anymore? The new way of doing this is very verbose :thinking:

Reply:
Sadly yes. I also don't like it:

See https://github.com/schmittjoh/serializer/blob/master/UPGRADING.md

> Removed deprecated method JsonSerializationVisitor::addData, use ::visitProperty(new StaticPropertyMetadata('', 'name', 'value'), 'value') instead

We could add `trait` or something to wrap this.